packages/mux-player/README.md

```diff
@@ -201,6 +246,7 @@ If you prefer to use the in-code MSE-based engine (currently hls.js) whenever po
 | `videoWidth` <sub><sup>Read only</sup></sub>  | Returns an unsigned integer value indicating the intrinsic width of the resource in CSS pixels, or 0 if no media is available yet.                                                                                                                                                                                  | `0`         |
 | `volume`                                      | Is a double indicating the audio volume, from 0.0 (silent) to 1.0 (loudest).                                                                                                                                                                                                                                        | `1`         |
 | `customDomain`                                | Is a `String` that assigns a custom domain to be used for Mux Video.                                                                                                                                                                                                                                                | `undefined` |
+| `controlsList`                                | Is a `DOMTokenList` similar to `classList` which offers a way to hide the controls elements/buttons that are being shown by the player. For more, see the section on [`controlslist`](#controlslist)                                                                                                                | `''`        |
```

Comment:
chore(blocker): If this is going to be read-only by design, we should mention that here somewhere.